File: emby-lib/src/main/java/us/nineworlds/serenity/emby/model/AbstractCrew.java
package us.nineworlds.serenity.emby.model;

import us.nineworlds.serenity.common.media.model.ICrew;

public abstract class AbstractCrew implements ICrew {

  private String tag;

  @Override public String getTag() {
    return tag;
  }

  @Override public void setTag(String tag) {
    this.tag = tag;
  }

}

File: jellyfin-lib/src/main/kotlin/us/nineworlds/serenity/jellyfin/model/AbstractCrew.kt
package us.nineworlds.serenity.jellyfin.model

import us.nineworlds.serenity.common.media.model.ICrew

abstract class AbstractCrew : ICrew {
    private var _tag: String? = null

    override fun getTag(): String? {
        return _tag
    }

    override fun setTag(tag: String?) {
        _tag = tag
    }

}